recipes/chimes-calculator/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "chimes-calculator" %}
+{% set version = "1.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rk-lindsey/chimes_calculator/archive/refs/tags/v1.0.0.zip
```

Only one place to bump the version
```suggestion
  url: https://github.com/rk-lindsey/chimes_calculator/archive/refs/tags/v{{ version }}.zip
```